process/incident_management.md

```diff
@@ -0,0 +1,263 @@
+* Catching incident occurrences through our automation (monitoring/alerting) before they are noticed by the customer
+* Avoiding re-occurrences of similar incidents
+* Minimizing the time to detect a potential incident (MTTA)
+
+## Roles and Responsibilities
+
+### Incident First Responder
+Any SRE investigating a cluster issue becomes this, when they notice a **problem with a cluster or a specific application**  which:
+* may be a **customer–facing cluster having an issue impacting (degradation or outage)**
+* or cause **imminent major outage.**
+
+A major outage can be defined as one where our platform is running out of error budget and we start failing our SLOs.
+
+SRE teams have the concept of an on-duty shift person who is responsible for answering to alerts and incidents while the rest of the team is focusing on engineering work. These persons will usually be First Responders. Most often a shift Primary or Secondary become First Responders while investigating ongoing cases/alerts.
+
+
+#### Initial Response
+Before doing any further troubleshooting, First Responder must, in order:
+
+1. **Create a bridge.**
+2. Using IM, **post a message in the relevant channels** with:
+    * link to bridge,
+    * short description of issue (which service, what’s failing),
+    * your IM handle.
+Only now can you continue debugging while describing the problem to people joining the bridge.
+
+#### 10 Minutes Later
+**If** 10 minutes after your announcement your **region lead or manager haven’t joined** the bridge and you haven't managed to recover the service:
+* halt any further technical work
+* **pick an Incident Commander** from among the SREs present.
+
+Only now can you continue debugging.
+
+#### Comments / Reasoning
+* **The First Responder Protocol is the most important part.**
+* It needs to be:
+  * **as short as possible** – since everybody needs to know it by heart,
+  * **as simple and unambiguous as possible** – because it needs to guarantee proper bootstrapping of the full Incident Management Process even by someone who’s still in training and accidentally stumbled upon a serious issue.
+
+### Incident Owner
+* The Incident Owner is by default the First Responder
+* The Incident Owner role is not handed over from one shift to the next and stays the same throughout the lifecycle of the incident unless the current owner doesn't feel confident or can't assume the role throughout the incident. In that case the role can be handed over and the handover needs to be fully documented and communicated to the wider team.
+* The incident ticket is assigned to the Incident Owner
+
+#### Responsibilities
+The main responsibility of the Incident Owner is to assume the complete ownership of the incident from a technical standpoint:
+
+* Understanding the course of the incident and its technical details from start to finish
+* Organizing and driving the PMR meeting in the 5 business days after recovery
```

Comment:
sometimes 5 business days is not enough, IMO, there's no need to hardcode any timelines.

Reply:
this is our official SD process commitments but it could be replaced in this version by X days